src/components/Card.js

```diff
@@ -6,6 +6,13 @@ import './style.css';
 export const Card = () => {
   const firstLoad = useRef(true);
   const [qrvalue, setQrvalue] = useState('');
+  const smallSize = 175;
```

Comment:
nit: Consider inverting the variable naming to `sizeSmall`, `sizeMedium` and `sizeLarge`. IMO this helps conceptually scope their context.

Reply:
Done. Thanks!